docs/product/explore/feature-flags/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Feature Flags"
+sidebar_order: 100
+description: "Learn how to set up and interact with Sentry's feature flag evaluation tracking and feature flag change tracking."
+---
+
+<Alert level="info" title="Currently in Beta">
+
+The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.
+
+</Alert>
+
+Link your external feature flag integrations with Sentry to provide feature flag insights right inside the Sentry UI. Linking one or more integrations will allow you to view recent flag evaluations in one place and identify potential suspect flags related to errors. In addition, Sentry will provide insights on feature flag updates relative to error event timelines.
+
+## Touchpoints
+
+Feature flag insights can be found by navigating to the Issue Details page for any error event. There are two places where you can interact with flag data:
+* Flag updates (changes, deletions, and additions) will appear as a series on the event and user volume chart.
+* Flag evaluations will appear in the Feature Flag section in Issue Details as a table, with "suspect" flag predictions highlighted in yellow.
+
+Learn more about how to interact with feature flag insights within the Sentry UI by reading the [Issue Details docs](/product/issues/issue-details/#feature-flags/).
```

Comment:
Hmm, this seems like something we should be saying right here on the product index!